pick.lic

```diff
@@ -79,15 +79,26 @@ class LockPicker
         next
       end
 
-      until 'not even locked' == bput("pick my #{box} blind", 'not even locked', 'Roundtime')
-        waitrt?
-        next unless DRSkill.getxp('Locksmithing') >= 30
+      locked=true
+      while locked
+        case bput("pick my #{box} blind", 'not even locked', 'Roundtime','Find a more appropriate tool')
+        when 'not even locked'
+          waitrt?
+          locked=false
```

Comment:
If they are the same, I would prefer `break` so there is one less variable around.